Adithya/HTMLbasicsremake/Pokemon .html

```diff
@@ -0,0 +1,47 @@
+<html>
+<head>
+    
+<title>Poke</title>    
+    
+</head>
+<body>
+    <h1>First Gen Pokemon</h1>
+<table border="1">
+<tr>
+    <th>Image</th>
+    <th>name</th>
+    <th>Type</th>
+    <th>Evolves Into</th>
+</tr>
+<tr>
+    <th><img src="001Bulbasaur.png"> </th>
+    <th>Bulbassaur</th>
+    <th>Grass/Poison</th>
+    <th><a href="http://pokemon.wikia.com/wiki/Ivysaur">Ivyasaur</a></th>
```

Comment:
Use `th` is used only in the first row. Use `td` for other rows.
`th` stands for table heading and `td` stands for table definition.